.github/workflows/test.yml

```diff
@@ -1,10 +1,10 @@
 name: test
 
 "on":
-  push:
-    branches: [main]
   pull_request:
     branches: [main]
+  workflow_call:
+    null
```

Comment:
Could you instead assign an empty dictionary to `workflow_call`. The use of `null` here is somewhat obscure, I think. Like this:
```yaml
  workflow_call: {}
```